core/rawdb/freezer.go

```diff
@@ -285,12 +291,42 @@ func (f *freezer) TruncateAncients(items uint64) error {
 	if atomic.LoadUint64(&f.frozen) <= items {
 		return nil
 	}
+	var frozen uint64
+	for _, table := range f.tables {
+		if err := table.truncateHead(items); err != nil {
+			return err
+		}
+		// Tables should be aligned, only check the first table.
```

Comment:
`only check the first table` I assumed by that you mean check it against `items`? In that case you could add a condition `if frozen != items`. Or maybe I misunderstood the comment

Reply:
Fixed, this is supposed to be deleted but I forgot to do it. Thanks for catching it!